packages/vuetify/src/install.ts

```diff
@@ -1,53 +1,40 @@
-import OurVue, { VueConstructor } from 'vue'
-import { VuetifyUseOptions } from 'types'
-import { consoleError } from './util/console'
-
-export function install (Vue: VueConstructor, args: VuetifyUseOptions = {}) {
-  if ((install as any).installed) return
-  (install as any).installed = true
-
-  if (OurVue !== Vue) {
-    consoleError('Multiple instances of Vue detected\nSee https://github.com/vuetifyjs/vuetify/issues/4068\n\nIf you\'re seeing "$attrs is readonly", it\'s caused by this')
-  }
+import { App } from 'vue'
+import Vuetify, { VuetifySymbol } from './framework'
+import { ComponentOrPack, VuetifyUseOptions } from 'types'
 
+export function install (app: App, args: VuetifyUseOptions = {}) {
   const components = args.components || {}
   const directives = args.directives || {}
 
-  for (const name in directives) {
-    const directive = directives[name]
+  for (const key in directives) {
+    const directive = directives[key]
 
-    Vue.directive(name, directive)
+    app.directive(key, directive)
   }
 
-  (function registerComponents (components: any) {
-    if (components) {
-      for (const key in components) {
-        const component = components[key]
-        if (component && !registerComponents(component.$_vuetify_subcomponents)) {
-          Vue.component(key, component as typeof Vue)
-        }
+  (function registerComponents (components: Dictionary<ComponentOrPack>) {
+    if (!components) return false
+
+    for (const key in components) {
+      const component = components[key]
+
+      // Check for and register subcomponents
+      if (
+        component &&
+        component.$_vuetify_subcomponents
+      ) {
+        registerComponents(component.$_vuetify_subcomponents)
+
+        continue
       }
-      return true
+
+      app.component(key, component)
     }
-    return false
+
+    return true
   })(components)
 
-  // Used to avoid multiple mixins being setup
-  // when in dev mode and hot module reload
-  // https://github.com/vuejs/vue/issues/5089#issuecomment-284260111
-  if (Vue.$_vuetify_installed) return
-  Vue.$_vuetify_installed = true
-
-  Vue.mixin({
-    beforeCreate () {
-      const options = this.$options as any
-
-      if (options.vuetify) {
-        options.vuetify.init(this, options.ssrContext)
-        this.$vuetify = Vue.observable(options.vuetify.framework)
-      } else {
-        this.$vuetify = (options.parent && options.parent.$vuetify) || this
-      }
-    },
-  })
+  const vuetify = new Vuetify(args)
```

Comment:
You're passing args here which is VuetifyUseOptions but the constructor takes UserVuetifyPreset?